### DISCUSSION TITLE
Help instaling Dashy with portainer

### DISCUSSION THREAD

**Discussion by witzker** [Lissy93/dashy#1325] [Help and Support]
Hi,
I'm totally new with all this server stuff and learning.
Can anyone pls. Assist in installing Dashy with my portainer running in an ESXi 7 VM with Alpine Linux.

Many THX helping a newbie to have some success.
I tried This but failed:
`localhost:~# docker run -d \
>   -p 8080:80 \
>   -v ~/my-conf.yml:/app/public/conf.yml \
>   --name my-dashboard \
>   --restart=always \
>   lissy93/dashy:latest
Unable to find image 'lissy93/dashy:latest' locally
latest: Pulling from lissy93/dashy
59bf1c3509f3: Pull complete
b616ac4a64bf: Pull complete
3b9e1e8ab9ce: Pull complete
3507ddbf3909: Pull complete
3483be38397f: Pull complete
de8bda25b9b2: Pull complete
9417ac7006b9: Pull complete
552265c90d5f: Pull complete
Digest: sha256:d15f1a5992f76243740c3b7470137a3eda2b57fa742296f90f45f1b67739cccc
Status: Downloaded newer image for lissy93/dashy:latest
764d6045ba67888a4f7a105e0efce38de7a9ecc24baf122e6c23723c23f3a9bd
docker: Error response from daemon: failed to create shim task: OCI runtime crea                                      te failed: runc create failed: unable to start container process: error during c                                      ontainer init: error mounting "/root/my-conf.yml" to rootfs at "/app/public/conf                                      .yml": mount /root/my-conf.yml:/app/public/conf.yml (via /proc/self/fd/6), flags                                      : 0x5000: not a directory: unknown: Are you trying to mount a directory onto a f                                      ile (or vice-versa)? Check if the specified host path exists and is the expected                                       type.
`

**Comment by CrazyWolf13** [comment]
Can you test by creating the file my-local-conf.yml via some text editor or via "$touch my-local-conf.yml" and then running again?

**Reply by gdeans** [reply]
You might also want to specify the full path rather than relying on ~/ since that seems to be resolving to /root which may not be what you actually want.  Also, I've had to let the container set for a long time before it starts rendering but ymmv

**Comment by CrazyWolf13** [comment]
Seems like a permission issue you are trying to mount into root user folder?

Please try another location on the host as bind volume path.
